modules/example/js/multistep-display-ajax.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @file
+ *
+ * Adds handling of click event for view in order to demonstrate Ajax actions
+ * for multistep selection display.
+ */
+
+(function ($) {
+
+  'use strict';
+
+  /**
+   * Attaches the behavior of the media entity browser view.
+   */
+  Drupal.behaviors.multiStepAjaxSelection = {
+    attach: function () {
+      jQuery('.view-id-files_entity_browser_ajax td.views-field-filename')
+        .parents('tr')
+        .once()
+        .click(function () {
+          var $row = $(this);
+          var $input = $row.find('input');
+
+          // Here is way, how to trigger loading of entity for multipstep
+          // display over ajax. On multistep display element, "load-entities"
+          // event should be triggered with value for entity that should be
+          // loaded. Value is in format: type:id - for example: file:3
+          $row.parents('form')
```

Comment:
We shouldn't make other entity browsers implement this sort of logic - everyone using the View widget should benefit from the multistep work.